interpreter/src/main/java/org/intocps/maestro/interpreter/Interpreter.java

```diff
@@ -116,6 +117,22 @@ public Value caseAIdentifierStateDesignator(AIdentifierStateDesignator node, Con
         return question.lookup(node.getName());
     }
 
```

Comment:
This function is used instead of `caseAArrayStateDesignator` correct?
If so, can we not make `caseAArrayStateDesignator` use this function instead of `caseAAssigmentStm`?
Otherwise we risk breaking other stuff relying on `caseAArrayStateDesignator`.